### PR TITLE
expose Lib from DyBin/DyLib

### PR DESCRIPTION
**Pull request by 6293**
example: https://github.com/AluVM/alure/pull/2

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -23,22 +23,6 @@ jobs:
         uses: actions-rs/cargo@v1
         with:
           command: check
-  features:
```
Comment (dr-orlovsky) [inline .github/workflows/build.yml]: Whats wrong with them? Why are you keep removing them? :)

**Review by dr-orlovsky — COMMENTED** [review]
LGTM. just one quesiton

File: .github/workflows/build.yml (hunk 1 above)
Comment (6293) [reply .github/workflows/build.yml]: this was the unrebased commit from https://github.com/AluVM/aluasm/pull/5. now it is gone

**Review by dr-orlovsky — COMMENTED** [review]
Sorry, just understood that this is AluAsm repo, not AluRE. The plan was to have programs in the runtime environment - the compiler has nothing to do with executables. Can you pls move it there?

**Comment by 6293** [comment]
sure. Lib within DyBin/DyLib being private, I had to define DyProg here. Let me expose it in this PR

**Review by dr-orlovsky — APPROVED** [review]
ACK 061dacdbab8bf5a980b0d447095c799c8bad5a9d

**Pull request merged**